### ISSUE TITLE
compile errors

### ISSUE THREAD

**Issue by Tsoccerguy3** [Pokechu22/WorldDownloader#144]
compiling stops at 1.12 and started with , gradlew clean build -x test . The packages build up to and including all of the 1.11.2 branch and then crash

> Task :1.12.0:genSrgs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':1.12.0:genSrgs'.
> net.minecraftforge.srg2source.rangeapplier.SrgContainer.readSrgs(Ljava/lang/Iterable;)Lnet/minecraftforge/srg2source/rangeapplier/SrgContainer;

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.9/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 3m 42s
218 actionable tasks: 120 executed, 98 up-to-date

**Comment by Pokechu22** [comment]
Can you run `./gradlew :1.12.0:genSrgs --stacktrace` please?  The error message there looks like it's probably a `NoSuchMethodError`, but I've got no idea where that would happen or why it'd happen without the stacktrace.

**Comment by Tsoccerguy3** [comment]
```
#################################################
         ForgeGradle 2.3-SNAPSHOT-6084129e
  https://github.com/MinecraftForge/ForgeGradle
#################################################
                 Powered by MCP
             http://modcoderpack.com
     by: Searge, ProfMobius, R4wk, ZeuX
     Fesh0r, IngisKahn, bspkrs, LexManos
#################################################

> Configure project :1.12.1-litemod
This mapping 'snapshot_20190923' was designed for MC 1.14.3! Use at your own peril.

> Configure project :1.12.2-litemod
This mapping 'snapshot_20190923' was designed for MC 1.14.3! Use at your own peril.

> Configure project :1.14.1
New Dep: net.minecraft:client:1.14.1_mapped_snapshot_20190923-1.14.3

> Configure project :1.14.2
New Dep: net.minecraft:client:1.14.2_mapped_snapshot_20190923-1.14.3

> Configure project :1.14.3
New Dep: net.minecraft:client:1.14.3_mapped_snapshot_20190923-1.14.3

> Configure project :1.14.4
New Dep: net.minecraft:client:1.14.4_mapped_snapshot_20190923-1.14.3

> Task :1.12.0:genSrgs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':1.12.0:genSrgs'.
> net.minecraftforge.srg2source.rangeapplier.SrgContainer.readSrgs(Ljava/lang/Iterable;)Lnet/minecraftforge/srg2source/rangeapplier/SrgContainer;

* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':1.12.0:genSrgs'.
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:110)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:77)
        at org.gradle.api.internal.tasks.execution.OutputDirectoryCreatingTaskExecuter.execute(OutputDirectoryCreatingTaskExecuter.java:51)
        at org.gradle.api.internal.tasks.execution.SkipUpToDateTaskExecuter.execute(SkipUpToDateTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.ResolveTaskOutputCachingStateExecuter.execute(ResolveTaskOutputCachingStateExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ValidatingTaskExecuter.execute(ValidatingTaskExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipEmptySourceFilesTaskExecuter.execute(SkipEmptySourceFilesTaskExecuter.java:101)
        at org.gradle.api.internal.tasks.execution.FinalizeInputFilePropertiesTaskExecuter.execute(FinalizeInputFilePropertiesTaskExecuter.java:44)
        at org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:91)
        at org.gradle.api.internal.tasks.execution.ResolveTaskArtifactStateTaskExecuter.execute(ResolveTaskArtifactStateTaskExecuter.java:62)
        at org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:59)
        at org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:54)
        at org.gradle.api.internal.tasks.execution.ExecuteAtMostOnceTaskExecuter.execute(ExecuteAtMostOnceTaskExecuter.java:43)
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:34)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.run(EventFiringTaskExecuter.java:51)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:46)
        at org.gradle.execution.taskgraph.LocalTaskInfoExecutor.execute(LocalTaskInfoExecutor.java:42)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:273)
        at org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareWorkItemExecutor.execute(DefaultTaskExecutionGraph.java:258)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:135)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker$1.execute(DefaultTaskPlanExecutor.java:130)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.execute(DefaultTaskPlanExecutor.java:200)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.executeWithWork(DefaultTaskPlanExecutor.java:191)
        at org.gradle.execution.taskgraph.DefaultTaskPlanExecutor$ExecutorWorker.run(DefaultTaskPlanExecutor.java:130)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
Caused by: java.lang.NoSuchMethodError: net.minecraftforge.srg2source.rangeapplier.SrgContainer.readSrgs(Ljava/lang/Iterable;)Lnet/minecraftforge/srg2source/rangeapplier/SrgContainer;
        at pokechu22.test.begradle.customsrg.GenSrgsWithCustomSupportTask.doTask(GenSrgsWithCustomSupportTask.java:162)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:73)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.doExecute(StandardTaskAction.java:46)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:39)
        at org.gradle.api.internal.project.taskfactory.StandardTaskAction.execute(StandardTaskAction.java:26)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:786)
        at org.gradle.api.internal.AbstractTask$TaskActionWrapper.execute(AbstractTask.java:753)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter$1.run(ExecuteActionsTaskExecuter.java:131)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:300)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor$RunnableBuildOperationWorker.execute(DefaultBuildOperationExecutor.java:292)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.execute(DefaultBuildOperationExecutor.java:174)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:90)
        at org.gradle.internal.operations.DelegatingBuildOperationExecutor.run(DelegatingBuildOperationExecutor.java:31)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeAction(ExecuteActionsTaskExecuter.java:120)
        at org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeActions(ExecuteActionsTaskExecuter.java:99)
        ... 31 more


* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 5.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/4.9/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 31s
2 actionable tasks: 1 executed, 1 up-to-date
```

**Comment by Pokechu22** [comment]
Thanks.  Looks like there was [a change to ForgeGradle 2](https://github.com/MinecraftForge/ForgeGradle/commit/41e1860bb7079848b8041227dcb13d991fcd3974) that updated SRG2Source, and [there was an SRG2Source change](https://github.com/MinecraftForge/Srg2Source/commit/3c92131ef74a11f3c2a469ac0d49aaaf28a46f5d#diff-549e4a163a3d513f5c4fa2ed51132492L140) that removed that method.  I've updated begradle to fix it.

As a side note, you mention that you've been building with `gradlew clean build -x test`; I don't think the `-x test` is necessary anymore.  I can't remember exactly when I suggested that, but I'm guessing there was an issue with unit tests (was it with native libraries?) that I've since fixed.
